src/components/Issues/IssueTable/IssueCrimeChart.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useRef, useEffect } from 'react';
+import billboard, { bar } from 'billboard.js';
+import 'billboard.js/dist/billboard.css';
+import service from '../../../service';
+
+interface IProps {
+  issueId: string;
+}
+
+function Chart(props: IProps): React.ReactElement {
+  const { issueId } = props;
+  const chartContainer = useRef(null);
+  useEffect(() => {
+    (async () => {
+      const res = await service.getCrimesCountByIssue(issueId);
+      const { crimes } = res.data;
+      billboard.generate({
+        data: {
+          x: 'x',
+          json: {
+            x: [
+              ...crimes.map((crime: any) => {
+                const date = new Date(+crime._id);
+                return `${date.getFullYear()}-${date.getMonth()}-${
+                  date.getDate() <= 9 ? `0${date.getDate()}` : date.getDate()
+                } ${date.getHours() <= 9 ? `0${date.getHours()}` : date.getHours()}:00`;
+              }),
+            ],
+            error: [...crimes.map((crime: any) => crime.count)],
```

Comment:
이 부분은 map의 결과인 array를 다시 분해해서 다시 array로 묶는 것 같은데, [ ... ] 는 빼도 되지 않을까요?
```suggestion
            x: crimes.map((crime: any) => {
                const date = new Date(+crime._id);
                return `${date.getFullYear()}-${date.getMonth()}-${
                  date.getDate() <= 9 ? `0${date.getDate()}` : date.getDate()
                } ${date.getHours() <= 9 ? `0${date.getHours()}` : date.getHours()}:00`;
              }),
            error: crimes.map((crime: any) => crime.count),
```